### ISSUE TITLE
Attempt to connect nonexistent signal in ProjectListFilter

### ISSUE THREAD

**Issue by frank-lesser** [godotengine/godot#19105]
**Godot version:**
master  commit v. 12.5 
from the extended log in my build

<https://github.com/godotengine/godot/commit/5b11d16f219ff5741365cda8569297c0d39fee7a>
`   At: ProjectListFilter::ProjectListFilter() - OptionButton
   At: Object::connect() - In Object of type 'OptionButton': Attempt to connect nonexistent signal 'item_selected' to method 'ProjectListFilter._filter_option_selected'

   At: Object::connect() - In Object of type 'OptionButton': Attempt to connect nonexistent signal 'size_flags_changed' to method 'ProjectListFilter.queue_sort'

   At: Object::connect() - In Object of type 'OptionButton': Attempt to connect nonexistent signal 'minimum_size_changed' to method 'ProjectListFilter._child_minsize_changed'

   At: Object::connect() - In Object of type 'OptionButton': Attempt to connect nonexistent signal 'visibility_changed' to method 'ProjectListFilter._child_minsize_changed'
`

**Comment by frank-lesser** [comment]
in ProjectListFilter::ProjectListFilter()  [project_manager.cpp] wrong signals are connected 
Object::connect just fails but the return value is normally not checked, so intention of the line 
	filter_option->connect("item_selected", this, "_filter_option_selected"); is to invoke _filter_option_selected but due to the non existent signal it will never executed 

connect return is never checked so it is better to log a warning inside connect which my build does

just needs to be fixed, maybe not the right place to post, sry

**Comment by vnen** [comment]
Is there any steps to reproduce?

**Comment by akien-mga** [comment]
>  master commit v. 12.5

What is `v. 12.5`? And steps to reproduce would indeed be good, as is it's a bit hard to follow.

**Comment by frank-lesser** [comment]
sry, that didn't appear in my latest builds, so it my have to do with my changes, 12.5 is 12 may the commit date of the master build I used

**Comment by reduz** [comment]
seems to be gone, so closing
